pkg/cli/sync.go

```diff
@@ -131,11 +119,16 @@ func (c *SyncCommand) Run(ctx context.Context, args []string) error {
 		return fmt.Errorf("unexpected arguments: %q", args)
 	}
 
-	if ok := slices.Contains(allowedSourceSystem, strings.ToUpper(c.source)); !ok {
+	// Convert source and destination to all caps so it matches the
+	// predefined system name const.
+	c.source = strings.ToUpper(c.source)
```

Comment:
In your usage and examples above you should use the proper case then, also, the usage string can include allowed values so users dont have to guess.

e.g. https://github.com/abcxyz/guardian/blob/main/pkg/platform/config.go#L49

Reply:
Added.